src/main/resources/data.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO employee(id, first_name, last_name, birth_date)
+VALUES
+(1, 'Anna', 'Nowak', '1985-06-15'),
+(2, 'Jan', 'Nowak', '1980-05-15'),
+(3, 'Kasandra', 'Cwir', '1987-06-07'),
+(4, 'Morimetr', 'Cwir', '1981-09-11');
+
```

Comment:
Jeżeli korzystamy z liquibase, to dane testowe też najlepiej wstawiać przy jego pomocy:
https://app.javastart.pl/kurs/spring/spring-data-profile-migracje/lekcja/spring21-dane-testowe-liquibase
Inaczej mogą być problemy z tym kiedy co się wykonuje (np. dane testowe spring może próbować wstawić zanim powstaną tabele itp.)